spec/request_handler/schema_parser_spec.rb

```diff
@@ -108,4 +108,12 @@ def run
     let(:error) { RequestHandler::SchemaValidationError }
     it_behaves_like 'handles invalid input data correctly'
   end
+
+  context 'data keys get symbolized when schema rules are symbols' do
```

Comment:
if this tests is supposed to trigger the deep symbolize keys bit then it doesn't work as expected. removing the deep symbolize keys line doesn't break this test